pkg/golinters/forbidigo.go

```diff
@@ -0,0 +1,63 @@
+package golinters
+
+import (
+	"sync"
+
+	"github.com/ashanbrown/forbidigo/forbidigo"
+	"github.com/pkg/errors"
+	"golang.org/x/tools/go/analysis"
+
+	"github.com/golangci/golangci-lint/pkg/golinters/goanalysis"
+	"github.com/golangci/golangci-lint/pkg/lint/linter"
+	"github.com/golangci/golangci-lint/pkg/result"
+)
+
+const forbidigoName = "forbidigo"
+
+func NewForbidigo() *goanalysis.Linter {
+	var mu sync.Mutex
+	var resIssues []goanalysis.Issue
+
+	analyzer := &analysis.Analyzer{
+		Name: forbidigoName,
+		Doc:  goanalysis.TheOnlyanalyzerDoc,
+	}
+	return goanalysis.NewLinter(
+		forbidigoName,
+		"Forbids identifiers",
+		[]*analysis.Analyzer{analyzer},
+		nil,
+	).WithContextSetter(func(lintCtx *linter.Context) {
+		s := &lintCtx.Settings().Forbidigo
+
+		analyzer.Run = func(pass *analysis.Pass) (interface{}, error) {
+			var res []goanalysis.Issue
+			linter, err := forbidigo.NewLinter(s.Forbid)
```

Comment:
Could you rename `linter` to `forbid` to avoid that the variable 'linter' collides with the imported package name?